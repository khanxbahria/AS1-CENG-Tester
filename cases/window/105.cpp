#include "../../your_code/Window.cpp"
#include "../../your_code/Browser.cpp"
#include "../../your_code/LinkedList.h"
int main() {
{
Window* wind = new Window();
wind->print();
Tab O = Tab("d", "p","ttt");
wind->newTab(O);
Tab X = Tab("ddd", "bb","r");
wind->isEmpty();
Tab T = Tab("e", "zzz","jjj");
wind->closeTab();
wind->changeActiveTabTo(5);
Tab M = Tab("i", "p","x");
wind->getActiveTab();
Tab E = Tab("ff", "p","ww");
wind->newTab(E);
Tab V = Tab("kkk", "xxx","iii");
wind->closeTab();
Tab B = Tab("y", "g","ii");
wind->closeTab();
wind->newTab(B);
Tab G = Tab("mmm", "rr","x");
wind->moveActiveTabTo(0);
Tab Z = Tab("o", "t","vv");
wind->print();
Tab Q = Tab("n", "pp","m");
wind->newTab(G);
Tab N = Tab("p", "kkk","xx");
wind->print();
Tab D = Tab("ppp", "ll","oo");
wind->getActiveTab();
Tab L = Tab("hhh", "vvv","n");
wind->closeTab();
wind->newTab(D);
Tab F = Tab("ee", "d","dd");
wind->closeTab();
wind->isEmpty();
wind->closeTab();
Tab S = Tab("kkk", "yy","d");
wind->changeActiveTabTo(5);
wind->getActiveTab();
wind->closeTab();
wind->print();
Tab W = Tab("lll", "q","lll");
wind->isEmpty();
Tab P = Tab("ff", "uuu","xxx");
wind->getActiveTab();
Tab A = Tab("u", "k","uuu");
wind->changeActiveTabTo(3);
Tab R = Tab("hhh", "xx","k");
wind->closeTab();
wind->closeTab();
wind->newTab(A);
Tab Y = Tab("cc", "j","rr");
wind->print();
wind->print();
wind->changeActiveTabTo(3);
Tab H = Tab("ccc", "r","ee");
wind->print();
Tab C = Tab("mmm", "gg","g");
wind->getActiveTab();
Tab K = Tab("aa", "i","p");
wind->changeActiveTabTo(1);
wind->newTab(C);
wind->getActiveTab();
wind->moveActiveTabTo(0);
Tab U = Tab("ff", "lll","j");
wind->getActiveTab();
wind->print();
wind->getActiveTab();
Tab I = Tab("i", "xxx","qqq");
wind->newTab(F);
wind->closeTab();
wind->getActiveTab();
wind->closeTab();
wind->newTab(H);
wind->getActiveTab();
wind->print();
wind->print();
wind->moveActiveTabTo(4);
wind->isEmpty();
wind->closeTab();
Tab J = Tab("dd", "gg","yyy");
wind->isEmpty();
wind->isEmpty();
wind->newTab(I);
wind->isEmpty();
wind->moveActiveTabTo(0);
wind->isEmpty();
wind->moveActiveTabTo(2);
wind->getActiveTab();
wind->changeActiveTabTo(1);
wind->moveActiveTabTo(5);
wind->closeTab();
wind->print();
wind->closeTab();
wind->isEmpty();
wind->print();
wind->newTab(J);
wind->isEmpty();
wind->isEmpty();
wind->print();
wind->newTab(K);
wind->print();
wind->closeTab();
wind->closeTab();
wind->moveActiveTabTo(2);
wind->newTab(L);
wind->newTab(M);
wind->closeTab();
wind->print();
wind->print();
wind->isEmpty();
wind->getActiveTab();
wind->isEmpty();
wind->print();
wind->print();
wind->isEmpty();
wind->closeTab();
wind->changeActiveTabTo(3);
wind->getActiveTab();
wind->print();
wind->print();
wind->closeTab();
wind->closeTab();
wind->changeActiveTabTo(5);
wind->newTab(N);
wind->newTab(P);
wind->print();
wind->getActiveTab();
delete wind;
}
	return 0;
}