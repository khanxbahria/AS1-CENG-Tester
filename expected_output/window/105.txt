The active tab is tab 0




The active tab is tab 0
g
y
ii

The active tab is tab 1
g
y
ii

rr
mmm
x

Window is empty
The list is empty.
The active tab is tab 0
k
u
uuu

The active tab is tab 0
k
u
uuu

The active tab is tab 0
k
u
uuu

The active tab is tab 0
gg
mmm
g

k
u
uuu

The active tab is tab 1
gg
mmm
g

r
ccc
ee

The active tab is tab 1
gg
mmm
g

r
ccc
ee

The active tab is tab 0
gg
mmm
g

Window is empty
The list is empty.
The active tab is tab 0
gg
dd
yyy

The active tab is tab 1
gg
dd
yyy

i
aa
p

The active tab is tab 0
vvv
hhh
n

The active tab is tab 0
vvv
hhh
n

The active tab is tab 0
vvv
hhh
n

The active tab is tab 0
vvv
hhh
n

Window is empty
The list is empty.
Window is empty
The list is empty.
The active tab is tab 1
kkk
p
xx

uuu
ff
xxx

